pkg/controller/core/workloads/containerizedworkload/containerizedworkload_controller_helper.go

```diff
@@ -32,6 +32,8 @@ func (r *Reconciler) renderDeployment(ctx context.Context,
 	}
 	// the translator lib doesn't set the namespace
 	deploy.Namespace = workload.Namespace
+	// make sure we don't have opinion on the replica count
+	deploy.Spec.Replicas = nil
```

Comment:
What if we don't use ManualScaler for an AppConfig?

Reply:
do we need to have an opinion on this field in other cases?